src/components/sourcepublisher/SourcePublisher.js

```diff
@@ -25,7 +25,7 @@ const typeToSchema = {
   MOLECULES: moleculesSchema,
   NEIGHBORHOODS: neighborhoodsSchema,
   RASTER: rasterSchema,
-  CELL_SETS: cellSetsSchema,
+  'CELL-SETS': cellSetsSchema,
```

Comment:
I find this part slightly odd, but if if seems like the clearest path to you, great!

Reply:
I think that ultimately this `"type"` field should not be converted to uppercase, but since that would be a breaking change, I think it should be left for after the portal data release / code freeze.